go.mod

```diff
@@ -0,0 +1,87 @@
+	github.com/juju/loggo v0.0.0-20180524022052-584905176618 // indirect
+	github.com/juju/testing v0.0.0-20180920084828-472a3e8b2073 // indirect
+	github.com/lestrrat/go-server-starter v0.0.0-20180220115249-6ac0b358431b
+	github.com/lib/pq v1.0.0 // indirect
+	github.com/martini-contrib/staticbin v0.0.0-20140517153916-b9631fb8c188
+	github.com/mattn/go-runewidth v0.0.2 // indirect
+	github.com/mattn/go-sqlite3 v1.9.0
+	github.com/mitchellh/hashstructure v0.0.0-20170609045927-2bca23e0e452
+	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
+	github.com/myesui/uuid v1.0.0 // indirect
+	github.com/nullbio/inflect v0.0.0-20170731032957-3edb4592a77b // indirect
+	github.com/olekukonko/tablewriter v0.0.0-20180506121414-d4647c9c7a84
+	github.com/onsi/ginkgo v1.6.0
+	github.com/onsi/gomega v1.4.2
+	github.com/op/go-logging v0.0.0-20160211212156-b2cb9fa56473
+	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/pkg/errors v0.8.0
+	github.com/prometheus/client_golang v0.9.2 // indirect
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90 // indirect
+	github.com/rcrowley/go-metrics v0.0.0-20180503174638-e2704e165165
+	github.com/robertkrimen/otto v0.0.0-20180617131154-15f95af6e78d
+	github.com/robfig/cron v0.0.0-20180505203441-b41be1df6967
+	github.com/serenize/snaker v0.0.0-20171204205717-a683aaf2d516
+	github.com/sirupsen/logrus v1.3.0 // indirect
+	github.com/soheilhy/cmux v0.1.4 // indirect
+	github.com/streamrail/concurrent-map v0.0.0-20160823150647-8bf1e9bacbf6
+	github.com/stretchr/testify v1.3.0 // indirect
+	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
+	github.com/twinj/uuid v1.0.0
+	github.com/tylerb/gls v0.0.0-20150407001822-e606233f194d
+	github.com/tylerb/is v2.1.3+incompatible // indirect
+	github.com/ugorji/go/codec v0.0.0-20190204201341-e444a5086c43 // indirect
+	github.com/vattle/sqlboiler v2.5.0+incompatible
+	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f
+	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
+	github.com/xeipuuv/gojsonschema v0.0.0-20160323030313-93e72a773fad
+	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
+	github.com/ziutek/telnet v0.0.0-20180329124119-c3b780dc415b // indirect
+	go.etcd.io/bbolt v1.3.2 // indirect
+	golang.org/x/crypto v0.0.0-20180904163835-0709b304e793
+	golang.org/x/net v0.0.0-20181201002055-351d144fa1fc
+	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4 // indirect
+	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c // indirect
+	google.golang.org/appengine v1.1.0 // indirect
+	google.golang.org/genproto v0.0.0-20180621235812-80063a038e33 // indirect
+	google.golang.org/grpc v1.13.0 // indirect
+	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
+	gopkg.in/sourcemap.v1 v1.0.5 // indirect
+	gopkg.in/yaml.v2 v2.2.2
+)
+
+replace github.com/onsi/ginkgo v1.6.0 => github.com/cloudwan/ginkgo v1.6.0-0.20190213151947-95174e8d10cd
```

Comment:
@zimnx please take a look at this. The goal was to point towards `gohan-v1.6.0` branch but I'm not sure if this was done correctly.

Reply:
Looks good.